docs/frameworks/symfony.md

```diff
@@ -6,90 +6,112 @@ introduction: Learn how to deploy serverless Symfony applications on AWS Lambda
 
 This guide helps you run Symfony applications on AWS Lambda using Bref. These instructions are kept up-to-date to be compatible with the latest Symfony version.
 
-A demo application is available on GitHub at [github.com/mnapoli/bref-symfony-demo](https://github.com/mnapoli/bref-symfony-demo).
+Multiple demo applications are available on GitHub at [github.com/brefphp/examples/Symfony](https://github.com/brefphp/examples/tree/master/Symfony).
 
 ## Setup
 
-Assuming we're inside an existing Symfony project, let's install Bref via Composer:
+First, make sure you have followed the [Installation guide](../installation.md) to create an AWS account and install the necessary tools.
+
+Next, in an existing Symfony project, install Bref and the [Symfony Bridge package](https://github.com/brefphp/symfony-bridge).
 
 ```
-composer require bref/bref
+composer require bref/symfony-bridge
 ```
 
-Now we can create a `serverless.yml` configuration file (at the root of the project) optimized for Symfony:
+If you are using [Symfony Flex](https://flex.symfony.com/), you're done. Flex will automatically run
+the [bref/symfony-bridge recipe](https://github.com/symfony/recipes-contrib/tree/master/bref/symfony-bridge/0.1) which will perform the following tasks:
+
+- Create a `serverless.yml` configuration file optimized for Symfony.
+- Add the `.serverless` folder to the `.gitignore` file.
+
+Otherwise, you can create the `serverless.yml` file manually at the root of the project, with the following configuration:
```

Comment:
Just an idea : Maybe we could link to the recipes repositiory for those not using Flex, that would avoid synching the recipe and the docs every time we need to do a change ?

Most projects will use Flex as choosing Bref is rarely something you do on a legacy/not updated project.

WDYT ?

Reply:
Indeed, this will be easier to maintain afterwards.

Reply:
I removed the configuration example and added an annotation to recipe with a direct link